example/lava-docker.job

```diff
@@ -0,0 +1,53 @@
+# Zephyr JOB definition for frdm-k64f
+device_type: 'frdm-k64f'
+job_name: 'zephyr docker'
+
+timeouts:
+  job:
+    minutes: 3
+  action:
+    minutes: 3
+  actions:
+    wait-usb-device:
+      seconds: 40
+
+priority: medium
+visibility: public
+
+actions:
+
+- test:
+    docker:
+        image: kumargala/ci-run
```

Comment:
I've published the docker image sources here:
https://github.com/galak/docker-ci-run

Meant to be minimal env to be able to run zephyr sanitycheck.